src/pages/workspace/WorkspacesListPage.js

```diff
@@ -200,9 +200,10 @@ class WorkspacesListPage extends Component {
                 )}
                 <FixedFooter style={[styles.flexGrow0]}>
                     <Button
+                        accessibilityLabel={this.props.translate('workspace.new.newWorkspace')}
                         success
                         text={this.props.translate('workspace.new.newWorkspace')}
-                        onPress={() => Policy.createWorkspace()}
+                        onPress={Policy.createWorkspace}
```

Comment:
Revert this line. This intended because the `onPress` callback will call the `createWorkspace` with a parameter event which will translate to `ownerEmail`.

onPress currently does not pass any parameter (not sure if this is intended though) so that may change.

Reply:
but didn't @roryabraham say so [here](https://expensify.slack.com/archives/C01GTK53T8Q/p1682377415283179?thread_ts=1682369641.777619&cid=C01GTK53T8Q) ?

Reply:
I didn't get what you mean here? what has `ownerEmail` got to do here? I might be missing some context

Reply:
`onPress={Policy.createWorkspace}`
This means whatever arguments are passed to `onPress` pass them to `createWorkspace`.

This is similar to:
`onPress={(e) => Policy.createWorkspace(e)}`

`e` here will be the `ownerEmail` which is what the `createWorkspace` accepts as first argument.